pkg/synthetics/synthetics_api_integration_test.go

```diff
@@ -796,7 +796,8 @@ func TestSyntheticsCertCheckMonitor_Basic(t *testing.T) {
 				Values: []string{"avocado"},
 			},
 		},
-		Domain:                            "https://www.google.com",
+		// do not add a "https://" to the domain; recent error handling in Synthetics throws an "invalid domain" error with "https://"
+		Domain:                            "www.google.com",
```

Comment:
With `https://www.google.com`

<img width="361" alt="image" src="https://github.com/newrelic/newrelic-client-go/assets/127438038/00e57c58-73e5-4594-9c03-007d8df8282c">

This error does not appear without the `https://`